src/evict/evict_lru.c

```diff
@@ -2579,11 +2579,11 @@ __verbose_dump_cache_single(WT_SESSION_IMPL *session, uint64_t *total_bytesp,
      * skipped it.
      */
     if (F_ISSET(dhandle, WT_DHANDLE_EXCLUSIVE))
-        return (__wt_msg(session, "  Opened exclusively. Cannot walk tree, skipping."));
+        return (__wt_msg(session, " handle opened exclusively, cannot walk tree, skipping"));
```

Comment:
Is there a reason we've dropped the capital letter, full stop here?

Reply:
Yes. We slice & dice messages as part of error output, including prepending the session's name field and appending error strings. Putting capitals and periods in the msg/err function calls doesn't look too awful, but the resulting appearance in the middle of output messages just looks ugly.